docs/getting-started/config-env.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Configure Enviroment Variables
+---
+
+# Set up your environment variables/secrets
+
+- Take a look at `Envfile`. This file lists all the `ENV` variables we use and provides a fake default for any missing keys. You'll need to get your own free [Algolia credentials](http://docs.dev.to/get-api-keys-dev-env/#algolia) to get your development environment running.
+- This [guide](http://docs.dev.to/get-api-keys-dev-env/) will show you how to get free API keys for additional services that may be required to run certain parts of the app.
```

Comment:
I think the link `http://docs.dev.to/get-api-keys-dev-env/` in these two lines (7 and 8) here is probably wrong now that you re-organized the docs, you should replace it with the new relative paths